nautobot/core/tests/test_utils.py

```diff
@@ -11,6 +11,7 @@
 from nautobot.core.api import utils as api_utils
 from nautobot.core.models import fields as core_fields, utils as models_utils
 from nautobot.core.utils import data as data_utils, filtering, lookup, requests
+from nautobot.circuits import models as circuits_models
```

Comment:
Alphabetic import order

Reply:
You mean `ci` doesn't come after `co`? :-) Will fix.